ext/opentelemetry-ext-jaeger/src/opentelemetry/ext/jaeger/__init__.py

```diff
@@ -0,0 +1,369 @@
+# Copyright 2018, OpenCensus Authors
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Jaeger Span Exporter for OpenTelemetry."""
+
+import logging
+import socket
+import typing
+
+from thrift.protocol import TBinaryProtocol, TCompactProtocol
+from thrift.transport import THttpClient, TTransport
+
+import opentelemetry.trace as trace_api
+from opentelemetry.ext.jaeger.gen.agent import Agent as agent
+from opentelemetry.ext.jaeger.gen.jaeger import Collector as jaeger
+from opentelemetry.sdk.trace.export import Span, SpanExporter, SpanExportResult
+
+DEFAULT_AGENT_HOST_NAME = "localhost"
+DEFAULT_AGENT_PORT = 6831
+DEFAULT_COLLECTOR_ENDPOINT = "/api/traces?format=jaeger.thrift"
+
+UDP_PACKET_MAX_LENGTH = 65000
+
+logger = logging.getLogger(__name__)
+
+
+class JaegerSpanExporter(SpanExporter):
+    """Jaeger span exporter for OpenTelemetry.
+
+    Args:
+        service_name: Service that logged an annotation in a trace.Classifier
+            when query for spans.
+        agent_host_name: (Optional) The host name of the Jaeger-Agent.
```

Comment:
I think it's better not to include "optional" in the arg description and let the existence of a default value speak for itself. I see some args with defaults but _without_ the "optional" description below, which suggests these are already out of sync.

Reply:
Totally agree.